modules/programs/neovim.nix

```
@@ -289,13 +302,14 @@ in {
         configure.customRC -> programs.neovim.extraConfig
     '';
 
-    programs.neovim.generatedConfigViml = neovimConfig.neovimRcContent;
-
     home.packages = [ cfg.finalPackage ];
 
+    programs.neovim.generatedConfigViml = neovimConfig.neovimRcContent;
+
     xdg.configFile."nvim/init.vim" = mkIf (neovimConfig.neovimRcContent != "") {
-      text = neovimConfig.neovimRcContent;
+      text = cfg.generatedConfigViml;
```

```suggestion
      text = neovimConfig.neovimRcContent;
```
To be consistent with the `mkIf` above